capstone_project/src/main/webapp/pages/mapsTemplate.html

```diff
@@ -20,6 +18,21 @@ <h1>Maps Feature</h1>
       <br>
       <div id="map"></div>
       <br>
+      <h3>Don't see your college?</h3>
+      </p>Submit your school's info below and it'll appear on the map!</p>
+      <form action="/school-data" method="POST">
+        <p>School Name:</p>
+        <input name="name-input" required>
+        <br/><br/>
```

Comment:
remove the / in these br tags and any other tag that is self closing. [ref](https://google.github.io/styleguide/htmlcssguide.html#Document_Type)